brainglobe_registration/registration_widget.py

```diff
@@ -60,10 +63,15 @@
     RegistrationParameterListView,
 )
 from brainglobe_registration.widgets.select_images_view import SelectImagesView
+from brainglobe_registration.widgets.similarity_metrics_widget import (
+    SimilarityWidget,
+)
 from brainglobe_registration.widgets.transform_select_view import (
     TransformSelectView,
 )
 
+from .similarity_metrics import find_best_atlas_slice
```

Comment:
Just for consistency, we should stick to absolute imports as that's being used throughout the package

```suggestion
from brainglobe_registration.similarity_metrics import find_best_atlas_slice
```